packages/extra/etcd/templates/hook/job.yaml

```diff
@@ -0,0 +1,84 @@
+{{- if .Chart.Version | semverCompare ">= 2.6.1" }}
```

Comment:
This check makes no sense, the .Chart.Version will always be more or equal `2.6.1`